site-new/news/2021-02-02-newsletter-2.mdx

```diff
@@ -0,0 +1,136 @@
+import { Tweet } from 'react-tweet'
+
+# Armeria Newsletter vol. 2
+
+<AnimatedLogo />
+
+## From the devs
+
+✨ [Armeria 1.4.0](/release-notes/1.4.0/) has been released.
+
+- [Consul](https://consul.io/) service discovery support
+  - [Consul Registration for Server](/docs/server/service-registration/#consul-based-service-registration-with-consulupdatinglistener)
+  - [Consul Service Discovery for Client](/docs/client/service-discovery/#consul-based-service-discovery-with-consulendpointgroup)
+- Protect a fresh node from getting too much traffic suddenly by smoothly ramping up the weight of
+  [Endpoint](typeplural) in an [EndpointGroup](type) using [EndpointSelectionStrategy#rampingUp()](type).
+- Ability to close a connection after handling a certain number of requests or after a certain period of time,
+  rather than keeping it open indefinitely.
+
+🏆 We nominated contributors for a reward!
+
+- From the last contributor reward in April 2019 to Armeria 1.3.0 release in December 2020,
+  Armeria got 63 contributors. 
+- Surprisingly, 52 of them were first-time contributors. 
+- As a token of appreciation, we've sent a thank you note with a gift. 
+- Thanks all for participating with us, and we're looking forward to seeing you keep it up with this year. 😄
+
+{/* truncate */}
+
+<div style={{width: '100%', display: 'flex', flexWrap: 'wrap'}}>
+  <div class="tweet-box" style={{marginRight: '1rem'}}>
+    <Tweet id="1337766539328712706" />
```

Comment:
This tweet cannot be found. Should I remove it, or is there an alternative?

Reply:
Unfortunately, the original post seems to have been deleted. Let's remove it.

Reply:
Applied in [693ccc9](https://github.com/line/armeria/pull/6278/commits/693ccc932036029649c6dbe5936a54deb999cea7)